src/main/java/com/offdk/play/web/SlackCommandParamHandler.java

```diff
@@ -28,7 +33,30 @@ public Object resolveArgument(MethodParameter param, ModelAndViewContainer conta
         text=94070
         response_url=https://hooks.slack.com/commands/1234/5678
      */
-    return new SlackCommand();
+
+        //List<String> list = request.getNativeRequest().split("{}")[1].split("&");
+        String[] arr = request.getContextPath().split("&");
+        HashMap<String, String> d = new HashMap<String, String>();  
+
+        for (String a : arr){
```

Comment:
Single letter variable again.

Also, I'd move the 

> String[] arr = request.getContextPath().split("&");

to be right above the for each, easier to follow for context